src/test/java/com/amazon/ion/impl/EncodingDirectiveCompilationTest.java

```diff
@@ -44,10 +48,13 @@
 import java.util.TreeMap;
 import java.util.function.Consumer;
 
+import static com.amazon.ion.BitUtils.bytes;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
```suggestion
import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.allOf;
```

Only if you want to use `allOf` below.